lib/expo/po/duplicate_translations_error.ex

```diff
@@ -31,17 +31,17 @@ defmodule Expo.PO.DuplicateMessagesError do
         """
         To merge the duplicates, run:
 
-          mix expo.msguniq #{file}
+          mix expo.msguniq #{file} -o #{file}
```

Comment:
Let's make this even more explicit, users will be copy-pasting this anyways.

```suggestion
          mix expo.msguniq #{file} --output-file #{file}
```